tasks/install_deb.yml

```diff
@@ -16,6 +16,7 @@
     warn: false
   register: deb_info
   changed_when: false
+  check_mode: no
```

Comment:
Please, use `check_mode: false`

Reply:
However, the documentation says to use the values yes or no:
https://docs.ansible.com/ansible/latest/user_guide/playbooks_checkmode.html#enforcing-or-preventing-check-mode-on-tasks